contribs/tzdataexport.py

```diff
@@ -0,0 +1,614 @@
+#!/usr/bin/env python3
+
+# Copyright 2010-2023 The Wazo Authors  (see the AUTHORS file)
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>
+
+"""tzdataexport is a tool to export current UTC offsets and daylight-saving
+   rules from the tz database's source files. Those are the same files used by
+   the time zone compiler (zic) to create tz binary files. The format of these
+   files are documented in the zic man pages.
+
+   The default text output format is made of 3-fields lines. The fields are,
+   in order, the timezone identifier, the offset from UTC in seconds, and the
+   daylight-saving rules. Each fields are space delimited.
+
+   The tool doesn't go great length in input file checking. If you pass it
+   garbage, you might as well receive garbage (i.e. meaningless exception).
+
+   More info about the tz database can be found here:
+   - http://www.twinsun.com/tz/tz-link.htm
+"""
+
+__version__ = "$Revision$ $Date$"
```

Comment:
You can remove the `__version__` variable, the values are not substituted by the current build system and it is unused